.gitignore

```diff
@@ -59,3 +59,10 @@ typings/
 
 # next.js build output
 .next
+
+<<<<<<< HEAD
```

Comment:
i believe this should not be here

Reply:
I am going to fix this after the standup.